### PR TITLE
JvRollout realigns controls on collapse/expand 

### PR DESCRIPTION
**Pull request by MHumm**
Implement suggested fix for already acknowledged issue 6332
[http://issuetracker.delphi-jedi.org/view.php?id=6332](http://issuetracker.delphi-jedi.org/view.php?id=6332)

### CODE REVIEW

File: jvcl/run/JvRollOut.pas [hunk 1]
```diff
@@ -1451,13 +1451,17 @@ procedure TJvCustomRollOut.CheckChildVisibility;
       FChildControlVisibility.Sorted := True;
     end;
 
+    DisableAlign;
```
Comment (baka0815) [inline jvcl/run/JvRollOut.pas]: You might want to use ``try`` ... ``finally`` or the align might stay disabled if an exception occurs.

File: jvcl/run/JvRollOut.pas [hunk 2]
```diff
@@ -1451,13 +1451,17 @@ procedure TJvCustomRollOut.CheckChildVisibility;
...
+
     for I := 0 to ControlCount - 1 do
       if (Controls[I] is TWinControl) and (TWinControl(Controls[I]).Visible) then
       begin
         FChildControlVisibility.AddObject(Controls[I].Name, Controls[I]);
         if CollapseCtrlsOnButton or (TWinControl(Controls[I]).Top > ButtonHeight) then
           TWinControl(Controls[I]).Visible := False;
       end;
+
+    EnableAlign := true;
```
Comment (obones) [inline jvcl/run/JvRollOut.pas]: This looks weird, either it's a method like below, or it's a property. Does this even compile?
And I agree with @baka0815, this must be placed inside a `try..finally` block

**Comment by baka0815** [comment]
@MHumm you surrounded one Diable/Enable with a try..finally, but not the other.
I also second @obones comment, it looks kinda weird and should be identical in both cases (probably without hte ``:= True`` part).

**Comment by MHumm** [comment]
Ok, I found these mistakes and fixed them. I also found another Disable/EnableAlign pair in that unit which I put into a try/finally block. I hope that's correct as well.

**Comment by baka0815** [comment]
@MHumm Why did you close and delete this?

**Comment by MHumm** [comment]
Ok, I'm still a bit new to pull requests. Seems like I mistook your "approval" for a "merge". And because of that I deleted my local branch in Git. What to do? Create another branch and redo these modifications? Or how to fix that?

**Comment by baka0815** [comment]
You should be able to restore (undelete) your branch.
If that's not possible via the GitHub interface you should be able to push your local branch.
If that's also not possible because you also removed this, you need to redo your changes.

**Comment by MHumm** [comment]
Ok, as it was deleted completely, I recreated it. Is #125 now and can hopefully be approved and merged in now.

**Pull request closed without merging**